src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1675,6 +1680,171 @@ def test_group_deployment_crosstenant(self, resource_group):
             self.check('resourceGroup', '{vm_rg}')
         ])
 
+        self.cmd('group deployment create -g {vm_rg} -n {dn1} --template-file "{tf}" --parameters SIG_ImageVersion_id={sig_id} NIC_id={nic_id} --aux-tenants "{aux_tenant}"', checks=[
```

Comment:
>group deployment create [](start = 18, length = 23)

I don't see test recordings for this. Is it not needed for this change?

Reply:
Because this case is not easy to write as rerecord, so now it is a live_only mode.